client/layout/index.jsx

```diff
@@ -206,6 +220,7 @@ export default connect( state => {
 	const oauth2Client = getCurrentOAuth2Client( state );
 	const wccomFrom = get( getCurrentQueryArguments( state ), 'wccom-from' );
 	const isEligibleForJITM = [ 'stats', 'plans', 'themes', 'plugins' ].indexOf( sectionName ) >= 0;
+	const isFrankenflow = startsWith( currentRoute, '/start/frankenflow' );
```

Comment:
Hoping to change this later on, when we start adding more frankenflow-specific stuff to the store